aio_pika/abc.py

```diff
@@ -38,7 +38,7 @@
 ConsumerTag = str
 
 MILLISECONDS = 1000
-ZERO_TIME = datetime.utcfromtimestamp(0)
+ZERO_TIME = datetime(1970, 1, 1)
```

Comment:
Might be better use timezone here too?

Reply:
This constant is used only here: https://github.com/mosquito/aio-pika/blob/2e63f3a8266353757e3a922b90a1602225110818/aio_pika/message.py#L61-L63

So it should stay as naive as another operand.

@mosquito concerning this part of code: is `*t[:7]` right? I guess it should be `*t[:6]` as in other functions in this module..

Reply:
```python
import calendar

@decode_expiration.register(time.struct_time) 
 def decode_expiration_struct_time(t: time.struct_time) -> float: 
     return calendar.timegm(t)
```

we should try to check something like this.

Reply:
@mosquito I tidied up these single-dispatch functions within separate commit.